.github/workflows/hotfix-release.yml

```diff
@@ -0,0 +1,65 @@
+name: Release a hotfix
+on:
+  workflow_dispatch:
+    inputs:
+      pr-number:
+        description: "Which PR should be released?"
+        required: true
+      confirm:
+        description: "Confirm that the PR is up to date with the last published release, with the specific additional changes that should be released as part of this hotfix and the relevant package patch versions bumped"
+        type: boolean
+        required: true
+      prerelease:
+        description: "Publish to the hotfix tag on NPM for testing?"
+        type: boolean
+        required: true
+jobs:
+  hotfix-release:
+    name: Create Draft PR
+    if: ${{ inputs.confirm == true }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
```

Comment:
would it make sense to check for user affiliation before checking out the repo?

Reply:
We could do? I'm not sure it'll make much difference though.